frontend/ui/data/strings.lua

```diff
@@ -7,16 +7,16 @@ S.DUAL_PAGES = _("Dual Pages")
 S.PAGE_CROP = _("Page Crop")
 S.FULL_SCREEN = _("Full Screen")
 S.ZOOM_DPI = _("Zoom (dpi)")
-S.PAGE_MARGIN = _("Margin")
+S.PAGE_MARGIN = _("Margins")
```

Comment:
This change looks a bit suspicious, what's it about?

Reply:
Asked you the question in https://github.com/koreader/koreader/issues/6122#issuecomment-624668715:
> Now includes Margin (should it be plural?) 

<kbd>![image](https://user-images.githubusercontent.com/24273478/81317284-f2b52380-908c-11ea-90a9-e7e1f285cf8d.png)</kbd>
It is plural in cre:
<kbd>![image](https://user-images.githubusercontent.com/24273478/81317309-fba5f500-908c-11ea-9489-6d79d7812422.png)</kbd>

Reply:
I'd say that it shouldn't be.

Reply:
Why?

Reply:
IIRC, margin is the four all at once (e.g., page margin), or a single one. Two/three of them (e.g., left & right margins) are plural.

Reply:
Because it's just "the margin." As @NiLuJe wrote.

Reply:
"The margin of a written page is the empty space on the side."

(There's empty space on all four sides.)

Reply:
Allright, english has the right to be strange.
So, even when talking about _its_ size, and not only as a general concept? _The page margin is small_ ?

Reply:
Yep!